server/package.json

```diff
@@ -22,11 +22,11 @@
     "twilio": "^3.33.2"
   },
   "devDependencies": {
-    "eslint": "^6.7.2",
```

Comment:
is there a reason why you took this out? I prefer to have ESLint installed locally, rather than having `npx` pull down an entire install each time I want to run the linter.

Reply:
Just so there is one fewer thing to keep up-to-date. They seem to release quite frequently (https://www.npmjs.com/package/eslint).

I'm OK with switching it back, but in that case we'll want to switch it in the `npm run lint` command as well to ensure that we're running it the same way in Travis. I just made a commit for that and you can let me know what you think.

Reply:
Ah yeah, fair. Maybe this would be an opportunity to try a more liberal semver? Or maybe we can leave that until our semver review of `package.json`

Reply:
Let's leave it for the other task for now.